tests/meta/workflow/conftest.py

```diff
@@ -113,6 +116,39 @@ def microsalt_qc_fail_lims_project() -> str:
     return "ACC11111_qc_fail"
 
 
+@pytest.fixture
+def metrics_file_failing_qc(
+    microsalt_qc_fail_run_dir_path: Path,
+    microsalt_qc_fail_lims_project: str,
+    tmp_path: Path,
+) -> Path:
+    """Return a metrics file that fails QC with corresponding samples in the database."""
```

Comment:
In what way does it fail?

Reply:
No idea, guessing that some of the metrics it contains do not pass the thresholds